shards/rust/src/types.rs

```diff
@@ -4327,14 +4327,14 @@ macro_rules! shenum {
         const $EnumValue:ident = $value:expr;
       )+
     }
-    struct $SHEnumInfo:ident { }
+    pub struct $SHEnumInfo:ident { }
```

Comment:
Better to just add a  `$vis_2:vis` and then reference it below as
`$vis_2 struct $SHEnumInfo {` since this only changes the matched pattern, this would change it so the defined  SHEnumInfo uses the same visibility that was put here.

Reply:
Should be resolved in [9fff79b](https://github.com/fragcolor-xyz/shards/pull/771/commits/9fff79be08e8761316c8b9ee887e10812da626a4)